projects/bitcoin-core/build.sh

```diff
@@ -47,13 +49,18 @@ CONFIG_SITE="$PWD/depends/$BUILD_TRIPLET/share/config.site" ./configure --enable
 
 make -j$(nproc)
 
-# Limit to a few targets as temporary workaround for https://github.com/google/oss-fuzz/pull/5699#issuecomment-831030305
-FUZZ_TARGETS=( 'process_messages' 'asmap' )
+WRITE_ALL_FUZZ_TARGETS_AND_ABORT="/tmp/a" "./src/test/fuzz/fuzz" || true
```

Comment:
Is this just for testing?
What's the purpose of these `|| true` here and on 42?

Reply:
Removed the one in line 42 by merging the upstream patch